scripts/linters/pylint_extensions_test.py

```diff
@@ -2915,3 +2915,61 @@ def test_usage_of_is_not_on_none_does_not_add_message(self):
         compare_node = if_node.test
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_compare(compare_node)
+
+
+class NonTestFilesFunctionNameCheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        super(NonTestFilesFunctionNameCheckerTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.NonTestFilesFunctionNameChecker)
+        self.checker_test_object.setup_method()
+
+    def test_function_def_for_test_file_with_test_only_adds_no_msg(self):
+        def_node = astroid.extract_node(
+            """
+            def test_only_some_random_function(param1, param2):
+                pass
+            """
+        )
+        def_node.root().name = 'random_module_test'
+        with self.checker_test_object.assertNoMessages():
```

Comment:
[optional] Add a newline above to present visual test blocks (setup, [action-assertion]) [In general, we have three blocks setup, action, & assertion but in this case, we can only create two blocks as the action-assertions are connected.]

Reply:
Thanks! I will remember that in future. Did the change for all the test functions.